LINUX/i40e_netmap_linux.h

```diff
@@ -470,7 +471,13 @@ i40e_netmap_rxsync(struct netmap_kring *kring, int flags)
 			}
 			ring->slot[nm_i].len = ((qword & I40E_RXD_QW1_LENGTH_PBUF_MASK)
 			    >> I40E_RXD_QW1_LENGTH_PBUF_SHIFT) - crclen;
-			ring->slot[nm_i].flags = slot_flags;
+
+			curr_slot_flag = slot_flags;
+			if (unlikely((staterr & (1<<I40E_RX_DESC_STATUS_EOF_SHIFT)) == 0 )) {
+				curr_slot_flag |= NS_MOREFRAG;
```

Comment:
Well, it's for you to decide. :)  
If you can merge it as is, it's great.
If you find the temporary variable distracting, I can remove it, I don't care that much.